services/distribution/src/test/java/app/coronawarn/server/services/distribution/objectstore/integration/S3PublisherTestIT.java

```diff
@@ -47,6 +47,17 @@
   @Autowired
   private S3Publisher s3Publisher;
 
+
+  @BeforeEach
+  public void setup() {
+    objectStoreAccess.deleteObjectsWithPrefix("");
+  }
+
+  @AfterEach
+  public void teardown() {
+    objectStoreAccess.deleteObjectsWithPrefix("");
```

Comment:
just my personal opinion on 'clean up after test' - what if something fails? How can you easily figure out, what went wrong, if all has been deleted AFTER the test execution? Of course to start always with a clean environment and a deterministic test setup, it makes sense to delete leftover BEFORE the test. But everything, which was created during the test execution, should be kept till the next run - shouldn't it?